### PR TITLE
Bugfix: check "Ready For Hearing" VACOLS box if hearing requested is a travel board hearing.

### PR DESCRIPTION
**Pull request by kierachell**
Fix for issue where "Ready for Hearing" VACOLS checkbox was not being checked when certification was completed.

connects #889 

We were checking the wrong column for hearing request data earlier. The column we were checking was "Hearing Action (BFHA)", which is null until the hearing is held, according to @cmgiven. This PR changes certification to use "Hearing Request (BFHR)" instead. We only check the "Ready For Hearing" checkbox if the hearing requested is a travel board hearing. 

Testing:
After merging, can one of @kierachell or @astewarttistatech validate this on UAT? Thanks!

### CODE REVIEW

**Comment by amprokop** [comment]
Wait, actually, should we be removing all instances to `hearing_type`? Hmm.

File: app/models/vacols/case.rb [hunk 1]
```diff
@@ -90,10 +90,12 @@ class InvalidLocationError < StandardError; end
     nil => { full_name: nil, short: nil }
   }.freeze
 
-  HEARING_TYPES = {
```
Comment (amprokop) [inline app/models/vacols/case.rb]: is it safe to remove this?

File: app/models/vacols/case.rb [hunk 2]
```diff
@@ -90,10 +90,12 @@ class InvalidLocationError < StandardError; end
...
+  HEARING_REQUEST_TYPES = {
     "1" => :central_office,
     "2" => :travel_board,
-    "6" => :video_hearing
```
Comment (amprokop) [inline app/models/vacols/case.rb]: is it safe to remove this?

File: app/models/form8.rb [hunk 3]
```diff
@@ -62,7 +62,7 @@ def assign_attributes_from_appeal(appeal)
       soc_date: appeal.soc_date,
       representative_name: appeal.representative_name,
       representative_type: appeal.representative_type,
-      hearing_requested: appeal.hearing_type ? "Yes" : "No",
```
Comment (amprokop) [inline app/models/form8.rb]: this change affects the prepopulation of the form 8.

File: app/services/appeal_repository.rb [hunk 4]
```diff
@@ -86,7 +86,7 @@ def self.set_vacols_values(appeal:, case_record:)
       soc_date: normalize_vacols_date(case_record.bfdsoc),
       form9_date: normalize_vacols_date(case_record.bfd19),
       ssoc_dates: ssoc_dates_from(case_record),
-      hearing_type: VACOLS::Case::HEARING_TYPES[case_record.bfha],
+      hearing_type: VACOLS::Case::HEARING_REQUEST_TYPES[case_record.bfhr],
```
Comment (amprokop) [inline app/services/appeal_repository.rb]: i'm not sure why we were using bfha in the first case. looks like it was just a bug, but @shanear and @cmgiven can either of you remember why this was here?

File: spec/services/appeals_repository_spec.rb [hunk 5]
```diff
@@ -119,7 +119,7 @@
           AppealRepository.normalize_vacols_date(7.days.ago),
           AppealRepository.normalize_vacols_date(6.days.ago)
         ],
-        hearing_type: :video_hearing,
+        hearing_type: :central_office,
```
Comment (amprokop) [inline spec/services/appeals_repository_spec.rb]: one artifact of this change is that there is no longer a separate video hearing category, so i had to update this spec.

File: app/services/appeal_repository.rb (hunk 4 above)
Comment (shanear) [inline app/services/appeal_repository.rb]: I'm not sure. Our VACOLS knowledge was weak at the time we added this.

File: app/models/vacols/case.rb (hunk 2 above)
Comment (cmgiven) [inline app/models/vacols/case.rb]: Yes, it doesn't exist in BFHR.

File: app/services/appeal_repository.rb (hunk 4 above)
Comment (cmgiven) [inline app/services/appeal_repository.rb]: I think it's a bug. It should basically never be set at the time of certification.

Can we change `hearing_type` to `hearing_request` or `hearing_request_type`? That would be more accurate to what the field indicates.

File: app/models/vacols/case.rb (hunk 1 above)
Comment (cmgiven) [inline app/models/vacols/case.rb]: We'll probably need at some point, but if it's no longer called anywhere, yeah, probably good to clear it out.

**Pull request merged**